src/util/notify.cpp

```diff
@@ -48,7 +48,7 @@ namespace big::notify
 		else
 		{
 			if (g.reactions.crash.notify)
-				g_notification_service.push_error("Protections", std::format("Blocked {} crash from unknown player", crash));
+				g_notification_service.push_error("Protections", std::vformat("BLOCKED_CRASH_UNKNOWN_PLAYER"_T.data(), std::make_format_args(crash)));
```

Comment:
`Protections` should already be in the translation repo called `PROTECTIONS`.

Should be an easy fix as the other guy said.